### PR TITLE
CSA-108/feedback

### PR DESCRIPTION
**Pull request by tymnik**
- FeedbackModal, FeedbackForm components and styles;
- api, endpoint, type, schema for Feedback;
- bugfix in AvatarInput.

### CODE REVIEW

File: src/api/feedback.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+import { FeedbackCredentials } from 'src/types';
+
+import { ENDPOINTS } from './constants';
+
+export const sendFeedback = async (credentials: FeedbackCredentials) => {
+  const response = await axios.post(ENDPOINTS.FEEDBACK, credentials);
+  return response.data;
```
Comment (VovaLyapota) [inline src/api/feedback.ts]: Make destructurization on 8 line

File: src/components/ui/AvatarInput/AvatarInput.tsx [hunk 2]
```diff
@@ -22,7 +25,9 @@ const AvatarInput = ({
   title,
   imagePath,
   setImagePath,
+  imageContainerStyles = '',
   buttonTitle = 'Upload',
+  buttonStyles = '',
```
Comment (VovaLyapota) [inline src/components/ui/AvatarInput/AvatarInput.tsx]: you can exclude = '', clsx resolve cases where there is no styles in props

File: src/api/feedback.ts (hunk 1 above)
Comment (tymnik) [reply src/api/feedback.ts]: done

File: src/components/ui/AvatarInput/AvatarInput.tsx (hunk 2 above)
Comment (tymnik) [reply src/components/ui/AvatarInput/AvatarInput.tsx]: done

**Review by VovaLyapota — APPROVED** [review]

File: src/schemas/constants.ts [hunk 3]
```diff
@@ -7,8 +7,12 @@ export const MESSAGES = {
   PASSWORD_REPEAT_REQUIRED: 'Please confirm your password',
   PASSWORD_MIN: 'Password must be at least 8 characters long',
   PASSWORD_MAX: 'Password must be no longer than 20 characters',
+  COMMENT_REQUIRED: 'Comment is required',
+  COMMENT_MIN: 'Comment must be at least 7 characters long',
+  COMMENT_MAX: 'Comment must be no longer than 230 characters',
```
Comment (StanislavKharkiv) [inline src/schemas/constants.ts:12]: Why we put constants in schemas folder? Better make constants file or folder separately

File: src/schemas/constants.ts [hunk 4]
```diff
@@ -7,8 +7,12 @@ export const MESSAGES = {
...
 };
 
 export const LIMITS = {
   PASSWORD: [8, 20],
+  COMMENT: [0, 1000],
 };
```
Comment (StanislavKharkiv) [inline src/schemas/constants.ts]: Why you decide use arrays? For me more readable use objects or variables, but if you like this one better ok.

And message with 0 letters looks a little short ))

**Review by StanislavKharkiv — APPROVED** [review]

**Pull request merged**